src/components/data/PathListAutomation.js

```diff
@@ -145,16 +151,10 @@ export default function PathListAutomation(props) {
     };
 
     const handlePathListCreation = (values, actions) => {
-        const {
-            selectedResources,
-            pattern,
-            dest,
-            fileName,
-            foldersOnly,
-        } = values;
+        const { selectedPaths, pattern, dest, fileName, foldersOnly } = values;
 
         const submission = {
-            paths: selectedResources?.map((resource) => resource?.path),
+            paths: selectedPaths,
```

Comment:
Ah no!  This file got updated while you were out and looks your rebase reverted those changes (this might be easier to see here in the PR compared to Fork?).  We'll need those old changes put back in while keeping your dialog related changes.

Reply:
I've added the previous changes back.